isisd/isis_te.c

```diff
@@ -311,25 +311,28 @@ DEFUN (show_isis_mpls_te_router,
        "Router information\n")
 {
 
-	struct listnode *anode;
+	struct listnode *anode, *nnode, *inode;
```

Comment:
I see lots of commands being turned into the `show isis vrf all XXX`

It's better to implement the vrf XX or vrf all command then have it loop over everything.  As that this code makes no sense in it's current format

Reply:
This is done for all the display commands in isis.

Reply:
I think we probably need to figure out what to do here ... If the show commands in is-is are getting unwieldy, maybe it's time to open a feature request to go through and make them "right?"